ru/lessons/specifics/plug.md

```diff
@@ -0,0 +1,235 @@
+---
+layout: page
+title: Plug
+category: specifics
+order: 1
+lang: ru
+---
+
+Если вы знакомы с Ruby, то можете думать о модулях Plug как о Rack с добавкой Sinatra. Они предоставляют спецификацию для компонентов веб-приложений и адаптеры для веб-серверов. Хотя Plug и не являются частью ядра Elixir, это официальный проект Elixir.
```

Comment:
This paragraph was rewritten as follows: ```Если вы знакомы с Ruby, то можете думать о Plug как о комбинации Rack и Sinatra. Во-первых, Plug это набор договорённостей и спецификаций, позволяющих создавать универсальные компонуемые модули, иcпользуемые в веб-приложениях. Во-вторых - адаптеры соединений для различных веб-серверов на платформе Erlang VM. Хотя Plug и не является частью ядра Elixir, это официальный проект Elixir.``` Plug description was taken from official documentation (https://hexdocs.pm/plug/readme.html). But I'm still not 100% sure that it's very clear though...